docs/schema/reference.md

```diff
@@ -669,10 +675,10 @@ In OCDS each contracting process can have only one tender stage. There are a num
 
 * When one planning process results in many tenders;
 * When a contract is awarded following two distinct, but related, tender processes, such as in national frameworks with locally run mini-competitions;
-* When a contract results in the award of sub-contracts - and those sub-contracts are also tracked using OCDS;
+* When a contract results in the award of subcontracts - and those subcontracts are also tracked using OCDS;
```

Comment:
The RelatedProcess reference needs updating per https://github.com/open-contracting/standard/issues/1713 so I haven't reviewed the changes here.